docs/affinityassistants.md

```diff
@@ -0,0 +1,116 @@
+<!--
+---
+linkTitle: "Affinity Assistants"
+weight: 405
+---
+-->
+
+# Affinity Assistants
```

Comment:
nit: can you make sure to use consistent capitalization for PVC, PipelineRun, and TaskRun?

Reply:
Updated